sdk/highlight-node/CHANGELOG.md

```diff
@@ -55,3 +55,9 @@
 
 - Adds a `Handlers.serverlessFunction` for use as a error wrapper in AWS Lambda.
 - Adds a `H.stop()` method for shutting down the SDK and flushing unsent data.
+
+## 3.1.1
```

Comment:
should be 3.1.2

Reply:
Does this look right?

https://github.com/highlight/highlight/pull/5905/commits/b2d95ed381f0f58b59653ebc281847e5b7f643ba

I figured I'd bump everthing to v3.1.2 to match `@highlight-run/node`.